src/components/Images.vue

```diff
@@ -250,6 +271,42 @@ export default {
     startRunningCommand(command) {
       this.imageOutputCuller = getImageOutputCuller(command);
     },
+    async deleteImages() {
+      const message = `Delete ${ this.imagesToDelete.length } ${ this.imagesToDelete.length > 1 ? 'images' : 'image' }?`;
+      const detail = this.imagesToDelete.reduce((prev, curr) => {
+        if (!this.isDeletable(curr)) {
```

Comment:
From my reading, `this.isDeletable(curr)` should always be `true`.

But I would replace the `reduce` with the simpler

```
this.imagesToDelete.map(image => `${ image.imageName }:${ image.tag }`).join("\n")
```

Reply:
That's a much more elegant refactor, thanks and good catch 😁

I think my original implementation required the `reduce`, but that is obviously no longer needed here.